Lab2/src/UI/View/ConsolePrinter.cs

```diff
@@ -0,0 +1,13 @@
+﻿using ConsoleTables;
+using System.Collections.Generic;
+
+namespace UI.View
```

Comment:
View?
Helpers better folder for this.

And make ConsoleHelpers class for this.

Reply:
Fixed